test/e2e/upgradetest/framework/framework.go

```diff
@@ -98,12 +108,30 @@ func (f *Framework) CreateOperator() error {
 								ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.name"}},
 							},
 						},
+						ReadinessProbe: &v1.Probe{
+							Handler: v1.Handler{
+								HTTPGet: &v1.HTTPGetAction{
+									Path: probe.HTTPReadyzEndpoint,
+									Port: intstr.IntOrString{Type: intstr.Int, IntVal: 8080},
+								},
+							},
+							InitialDelaySeconds: 3,
```

Comment:
No particular reason. Anywhere between 3-5s seemed reasonable since my stress tests were passing earlier with this. Did you want to make it longer?